.circleci/config.yml

```diff
@@ -1,24 +1,82 @@
 ---
 version: 2
 jobs:
-  docs-lint:
+  lint:
     docker:
-      - image: quay.io/freedomofpress/circleci-docker:latest
+      - image: gcr.io/cloud-builders/docker
```

Comment:
Heyy @tmc one change I just noticed - is there any reason we shouldnt peg this image to something? I realize the previous was on `latest` which isnt best practice .. but we did control that image. If we move to an image we no longer control - perhaps we should peg to something? Says here that latest is currently on `17.06.1` according to https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/docker .